docs/source/community/build_ci_governance.rst

```diff
@@ -1,3 +1,5 @@
+:orphan:
```

Comment:
What does this tag mean?

Reply:
this is needed to suppress warnings in Sphinx build for pages that are not included in any toctree:
More info here: https://www.sphinx-doc.org/en/master/usage/restructuredtext/field-lists.html#special-metadata-fields

Reply:
Why are these not in any toctree? The left bar is not a toctree?

Reply:
they are not in toctree. They are in the right navbar added through a theme variable: https://github.com/pytorch/pytorch/pull/149331/files#diff-2457c157704d9b9c45968887f10da4d5e44f5193cab316cffcb8fbcf10ec2aa2R214